### DISCUSSION TITLE
git clone etc?

### DISCUSSION THREAD

**Discussion by shortduck** [pcottle/learnGitBranching#990] [General]
Hello,

I have just started learning on https://learngitbranching.js.org/, but the every first command is taught is git commit. I was thinking that the tutorial will start with git clone, or some basic introduction. Am I mi49ssing something? Or does this tool assumes that the user will know some git commands already?

Thanks

**Comment by pcottle** [comment]
It's somewhat intentional; you can use git locally on your filesystem before interacting with a remote git repository. It's a bit simpler to focus on just your local state before getting into pushing and pulling

**Reply by waldyrious** [reply]
Not to play the devil's advocate here, but following that logic, shouldn't the first command to be taught be `git init`?

**Reply by pcottle** [reply]
Arguably yes, it's just that we don't really have a way to show that you're in an empty folder 🤷🏼‍♂️ 

For what it's work, this project has been running for close to 10 years and this is the first time this feedback has come up (from my memory)

**Comment by chrisxaustin** [comment]
I've been using git for a long time and always recommend this site to people new to git (and sometimes people who are not new, but could use some tips). The remote branch tutorial surprised me since I would have expected it to start with `git init` instead of `git clone`. There was a note about that in the text for the lesson, but it felt off to me. I'd also prefer a vertical swimlane to separate the local and remote git, vs using dashes borders.
